package.json

```diff
@@ -26,9 +26,12 @@
   },
   "homepage": "https://github.com/Khan/aphrodite",
   "devDependencies": {
-    "babel": "^5.8.23",
-    "babel-core": "^5.8.25",
-    "babel-loader": "^5.3.2",
+    "babel": "^6.5.2",
```

Comment:
FWIW, I ended up doing a similar update in https://github.com/Khan/aphrodite/pull/281